### ISSUE TITLE
input error .tif (pit remove tool)

### ISSUE THREAD

**Issue by Pauli999** [dtarb/TauDEM#128]
I have some problems relating to the input of the elevation grid (tif format).
Took the steps of the tutorial ´QUICK START GUIDE TO USING THE TAUDEM ARCGIS TOOLBOX´ to extract the grid cubdem to tif format. 
But if I want to use the grid in tiff format as input for the pit remove tool for example it gives an error.

The input error also occurs when I want to use the connectivity index (Cavalli) tool, which also requires a tif format.

Can somebody help me with this problem please?

![printscreeb](https://cloud.githubusercontent.com/assets/28714466/26118957/03000650-3a42-11e7-9d64-f8a5ef6b1fb0.png)

**Comment by dtarb** [comment]
Did you sort this out?

**Comment by Pauli999** [comment]
I did, thanks for asking. It was probably a problem caused by the
installation proces of the software. I downloaded arcgis again and the
error did not show up anymore.


Op 18 mei 2017 23:33 schreef "David Tarboton" <notifications@github.com>:

Did you sort this out?

—
You are receiving this because you modified the open/close state.
Reply to this email directly, view it on GitHub
<https://github.com/dtarb/TauDEM/issues/128#issuecomment-302592727>, or mute
the thread
